BrowserKit/Sources/ToolbarKit/AddressToolbar/BrowserAddressToolbar.swift

```diff
@@ -21,6 +21,8 @@ public class BrowserAddressToolbar: UIView, AddressToolbar, ThemeApplicable, Loc
         static let actionSpacing: CGFloat = 0
         static let buttonSize = CGSize(width: 44, height: 44)
         static let locationHeight: CGFloat = 44
+        // This could be changed at some point, depending on the a11y UX design.
+        static let locationMaxHeight: CGFloat = 54
```

Comment:
Should we maybe grow the height using `UIFontMetrics.default.scaledValue(for: UX.locationHeight)` but use `UX.locationHeight` as min height instead of setting a max height? That (and also the max height) should be discussed with design.